packages/astro/playwright.config.js

```diff
@@ -27,16 +27,6 @@ const config = {
 		/* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
 		trace: 'on-first-retry',
 	},
-	projects: [
-		{
-			name: 'Chrome Stable',
-			use: {
-				browserName: 'chromium',
-				channel: 'chrome',
-				args: ['--use-gl=egl'],
-			},
-		},
-	],
```

Comment:
Why was this removed?

Reply:
I need this to run E2E tests locally. It shouldn't have been committed. I'll undo it.